include/nbl/video/utilities/CPropertyPool.h

```diff
@@ -43,13 +43,20 @@ class CPropertyPool final : public IPropertyPool
         static inline core::smart_refctd_ptr<this_t> create(ILogicalDevice* device, const uint32_t capacity, const bool contiguous = false)
         {
             asset::SBufferRange<video::IGPUBuffer> blocks[PropertyCount];
-            for (auto i=0u; i<PropertyCount; i++)
+            video::IGPUBuffer::SCreationParams params;
+            params.queueFamilyIndexCount = 0u;
+            params.queueFamilyIndices = nullptr;
+            params.sharingMode = asset::ESM_CONCURRENT;
+            //params.usage = // should be user-provided probably
```

Comment:
STORAGE_BUFFER first and foremost, you can take additional usages from the user and `|` them in